src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,10 @@
+package seedu.addressbook.data.person;
+
+/**
+ * An interface to print contact details.
+ */
+
+public interface Printable {
+
+    String getPrintableString();
```

Comment:
think of a suitable class where this method can to be implemented to return the printable string for all members passed.